docker/bin/entrypoint.sh

```diff
@@ -5,15 +5,22 @@ set -e
 HASSIO_OPTIONSFILE=/data/options.json
 
 if [ -f ${HASSIO_OPTIONSFILE} ]; then
-    CONFIG=$(grep config_file ${HASSIO_OPTIONSFILE}| cut -d ':' -f 2 | sed s/[\"}]//g )
+    CONFIG=$(grep -o '"config_file": "[^"]*' ${HASSIO_OPTIONSFILE} | grep -o '[^"]*$')
```

Comment:
Hier musste ich das bestehende Konstrukt ändern, da sonst der Start mit mehr als einer Option in /data/options.json failed:
```
# cat /mnt/data/supervisor/addons/data/49686a9f_evcc/options.json
{
  "config_file": "/config/evcc.yaml",
  "sqlite_file": "/data/evcc.db"
}
```
Vorher:
```
# grep config_file /mnt/data/supervisor/addons/data/49686a9f_evcc/options.json  | cut -d ':' -f 2 | sed s/[\"}]//g
 /config/evcc.yaml,
```

Nachher:
```
# grep -o '"config_file": "[^"]*' /mnt/data/supervisor/addons/data/49686a9f_evcc/options.json | grep -o '[^"]*$'
/config/evcc.yaml
```